src/components/ViewEditor.js

```diff
@@ -19,6 +19,29 @@ const DEFAULT_STATE = {
 	fileType: FILE_TYPE.EMPTY
 };
 
+export const handleSaveViewSpec = async () => {
+	FileStorageSystemClient.doSaveViewSpec(
+		store.getState().tagsInView,
```

Comment:
not sure what the odds of this, but maybe we should call `getState` once in case, the user tries to switch away from the view midway through save where the following fields may be from another doc?

Reply:
👍